samples/textDetection.js

```diff
@@ -15,10 +15,9 @@
 
 'use strict';
 
-const async = require('async');
 const fs = require('fs');
 const path = require('path');
-
+const {promisify} = require('util');
```

Comment:
line 24. the link is not working, and please add a full stop (period) for all the comments with complete sentences as some of them have full stops (line 33), and others don't.

Reply:
Done! 
comments from line-21 to line-23 is single sentence, so not added full stop (period) there, elsewhere it's added.
link update to https://cloud.google.com/docs/authentication/getting-started, this page speaks as per the comments.

https://github.com/vijay-qlogic/nodejs-vision/pull/1/commits/bb6ccacc6481592a693e8d7c357219f88de87c96